packages/server/src/api/epicRouter.ts

```diff
@@ -133,11 +132,17 @@ export const buildEpicRouter = (
             variant.tickerSettings && tickerData.addTickerDataToSettings(variant.tickerSettings);
         const showReminderFields = variant.showReminderFields ?? getReminderFields();
 
+        // HERE - limit to a given regional choiceCardAmounts
+        const contributionAmounts = choiceCardAmounts.get();
+        const requiredRegion = countryCodeToCountryGroupId(targeting.countryCode ?? 'GB');
+        const mvtId = targeting?.mvtId ?? 0;
```

Comment:
beyond the scrope of this PR, but we should really make mvtId mandatory in the targeting payload - we can't have AB testing without it!

Reply:
I've created a new Trello card for this in the Engineering Backlog